cfn-resources/x509-authentication-database-user/test/README.md

```diff
@@ -0,0 +1,40 @@
+## MongoDB::Atlas::X509AuthenticationDatabaseUser
+
+### Impact
+The following components use this resource and are potentially impacted by any changes. They should also be validated to ensure the changes do not cause a regression.
+- X509 authentication database user L1 CDK constructor
+
+
+
+
+### Resources (and parameters for local tests) needed to manually QA:
+CustomerX509 is to be manually provided.
+- Atlas project (created by cfn-test-create-inputs.sh)
+- CustomerX509
+
+
+
+## Manual QA:
+
+### Prerequisite steps:
+1. Create a test certificate: One of the ways to create a test certificate is to install OpenSSL and run the following command (source):
+```
+openssl req -new -newkey rsa:1024 -days 365 -nodes -x509 -keyout test.key -out test.cert
+```
+A certificate for you will be generated in test.cert file.
+2. Use cert from #1 and update cert in inputs/inputs_1_create.json before running cfn test and when creating the stack in AWS CloudFormation in the steps below.
+
+
+### Steps to test:
+1. Follow general [prerequisites](../../../TESTING.md.md#prerequisites) for testing CFN resources.
+2. Follow [general steps](../../../TESTING.md.md#steps) to test CFN resources.
+
+### Success criteria when testing the resource
+1. X.509 Authentication should be enabled correctly set up in your Atlas Project as per configuration specified in the inputs/example. This can be found under Security section -> Advanced:
+
+<img src="https://user-images.githubusercontent.com/122359335/227374480-1afa48a4-5265-4a2a-ad92-067f5015eeca.png">
+2. General [CFN resource success criteria](../../../TESTING.md.md#success-criteria-when-testing-the-resource) should be satisfied.
```

Comment:
this link does not seem to be rendered corrently

Reply:
@andreaangiolillo the link for the image?

Reply:
this one 

[CFN resource success criteria](../../../TESTING.md.md#success-criteria-when-testing-the-resource)

Reply:
![Screenshot 2023-03-24 at 10 18 23](https://user-images.githubusercontent.com/5663078/227494744-4662aafd-e79a-48d8-b3d3-4e430ad99440.png)

Reply:
ah, okay nvm my comment then